testing/testmedia.cpp

```diff
@@ -2517,16 +2517,14 @@ class UdpSource: public virtual Source, public virtual UdpCommon
     {
         bytevector data(chunk);
         sockaddr_any sa(sadr.family());
-        socklen_t si = sa.size();
         int64_t srctime = 0;
-        int stat = recvfrom(m_sock, data.data(), (int) chunk, 0, sa.get(), &si);
+        int stat = recvfrom(m_sock, data.data(), (int) chunk, 0, sa.get(), &sa.syslen());
```

Comment:
Dereferencing a temporal object `SysLenWrapper::syslen`.... 🤔

Reply:
Allowed, as long as you don't access it after it is deleted. We know it is not deleted until the current instruction finishes execution, so it won't happen before returning from the function. It actually does the same as the "temporary variable" solution used previously, just the trick is hidden inside.